.github/workflows/build_and_test.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         run: |
           apk update && apk add cppcheck python3-dev
           python3 -m pip install cmake-format
+          pip install clang-format==18.1.6
```

Comment:
Concatenate in one line

python3 -m pip install cmake-format clang-format==18.1.6

Reply:
thanks, fixed.